daemon/src/notif.rs

```diff
@@ -0,0 +1,15 @@
+use notify_rust::Notification;
+
+/**
+ * This is just a thin wrapper around the Notification stuff for now.
+ * The library says it always returns Ok, so we're not worrying about anything
+ * else at the moment.
+ */
+pub fn notif(summary: &str, msg: &str) {
+    match Notification::new().summary(summary).body(msg).show() {
```

Comment:
Also, if you want to get the same message but without the empty okay, use the expect method.
```rs
Notification::new().summary(summary).body(msg).show().expect("Unable to display notification!")
```